pgjdbc/src/main/java/org/postgresql/PGProperty.java

```diff
@@ -427,6 +427,17 @@ public enum PGProperty {
       "5432",
       "Port of the PostgreSQL server (may be specified directly in the JDBC URL)"),
 
+  /**
+   * Controls the styles of parameter placeholders accepted in prepared statements. The default value of {@code jdbc}
+   * disables non-standard placeholders.
+   */
+  PLACEHOLDER_STYLE(
+      "placeholderStyle",
+      "jdbc",
+      "Specifies which extra non-standard styles of placeholders are to be recognized by the driver, if any. None will disable parameters in Prepared Statements",
+      false,
+      new String [] {"any", "jdbc", "named", "native", "none"}),
```

Comment:
What would be the use case for having `placeholderStyle=none`?
I think it makes the usage confusing